upflux-client/src/App.tsx

```diff
@@ -1,24 +1,26 @@
 import logo from "./assets/logos/logo-dark.png";
 import './App.css';
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'; 
-import { Navbar } from './features/Navbar/Navbar'; 
-import { Header } from './features/Header/Header'; 
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { Navbar } from './features/Navbar/Navbar';
+import { Header } from './features/Header/Header';
 import { About } from './features/About/About';
 import { ContactUs } from './features/ContactUs/ContactUs';
 import { Footer } from './features/Footer/Footer';
-import { LoginComponent } from './features/Login/Login'; // Import the LoginComponent
-import { MantineProvider } from "@mantine/core";
+import { LoginComponent } from './features/Login/Login';
+import { MantineProvider, Container } from "@mantine/core";
+import { PasswordSettingsContent } from "./features/PasswordSettings/PasswordSettings";
 
 export const App = () => {
   return (
     <MantineProvider>
-      <Router> 
+      <Router>
         <Routes>
-          <Route 
-            path="/" 
+          {/* Home route */}
+          <Route
+            path="/"
             element={
               <>
-                <Navbar />
+                <Navbar home={true} />
```

Comment:
`home` doesnt provide any meaning to tell us what it is please change it to completing more meaningful. Thanks